gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/WriteModuleXml.kt

```diff
@@ -151,10 +164,20 @@ open class WriteModuleXml @Inject constructor(_objects: ObjectFactory) : Default
                     }
                 }
 
```

Comment:
We might want to trap and fail on `moduleDependencies.isPresent && moduleDependencySpecs.isPresent`  to shunt module devs in one direction or another. That'd be a hard guardrail.

Or we could allow both, and turn the `else if` below into a regular `if`. Wild west stuff, but still reasonable.

Or we could leave this conditional as is until somebody tries to use both and their `moduleDependencySpecs` are silently ignored, and then fix it the way they think it should work.

Reply:
Hmm alternatively I could just flip the conditionals so if both are present we use the newer one